### PR TITLE
[15.0][MIG]hotel_restaurant

### PR DESCRIPTION
**Pull request by ShwetaSerpentCS**

### CODE REVIEW

File: hotel_restaurant/LICENSE/LICENSE [hunk 1]
```diff
@@ -0,0 +1,16 @@
+    OpenERP, Open Source Management Solution
+    Copyright (C) 2004-2010 OpenERP SA (<http://www.openerp.com>)
+    Copyright (C) 2021-TODAY Serpent Consulting Services Pvt. Ltd. (<http://www.serpentcs.com>).
```
Comment (uncannycs) [inline hotel_restaurant/LICENSE/LICENSE]: Year should be 2022

File: hotel_restaurant/LICENSE/LICENSE [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Affero General Public License as
```
Comment (uncannycs) [inline hotel_restaurant/LICENSE/LICENSE]: Can we use a V15 level licence file?

File: hotel_restaurant/README.rst [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fvertical--hotel-lightgray.png?logo=github
+    :target: https://github.com/OCA/vertical-hotel/tree/14.0/hotel_restaurant
```
Comment (uncannycs) [inline hotel_restaurant/README.rst]: version should be v15

File: hotel_restaurant/README.rst [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+    :alt: OCA/vertical-hotel
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/vertical-hotel-14-0/vertical-hotel-14-0-hotel_restaurant
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/157/14.0
```
Comment (uncannycs) [inline hotel_restaurant/README.rst]: Check entire this file and make version changes from 14.0 to 15.0

File: hotel_restaurant/__manifest__.py [hunk 5]
```diff
@@ -0,0 +1,26 @@
+# See LICENSE file for full copyright and licensing details.
+
+{
+    "name": "Hotel Restaurant Management",
+    "version": "15.0.1.0.0",
```
Comment (uncannycs) [inline hotel_restaurant/__manifest__.py:6]: Initial version should be start from the 0

**Review by uncannycs — CHANGES_REQUESTED** [review]
Fix mention comments and good to go

**Comment by JayVora-SerpentCS** [comment]
@ShwetaSerpentCS pls update.

File: hotel_restaurant/__manifest__.py (hunk 5 above)
Comment (ShwetaSerpentCS) [reply hotel_restaurant/__manifest__.py:6]: @uncannycs I followed version based on OCA Guidelines https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#version-numbers

**Comment by ShwetaSerpentCS** [comment]
@JayVora-SerpentCS @murtuzasaleh This PR is ready for review.

**Comment by ShwetaSerpentCS** [comment]
> @JayVora-SerpentCS @murtuzasaleh This PR is ready for review.

@nikul-serpentcs

**Pull request merged**